### PR TITLE
Week2 homework

### PR DESCRIPTION
**Pull request by Nadoshka**

### CODE REVIEW

File: homework/index.js [hunk 1]
```diff
@@ -30,18 +33,82 @@
...
+    const root = document.getElementById('root');
+    createAndAppend('div', root, { text: error.message, class: 'alert-error' });
+  }
+
+  function app(repos) {
+    const root = document.getElementById('root');
+    const container = createAndAppend('div', root, {
+      class: 'container',
+    });
+    const header = createAndAppend('header', container, {
```
Comment (Meno-101) [inline homework/index.js]: Nice using of HTML5 element

File: homework/index.js [hunk 2]
```diff
@@ -30,18 +33,82 @@
...
+      createAndAppend('li', listInfo, {
+        text: `Updated: ${repos[select.value].updated_at}`,
+      });
+      const contributorsUrl = repos[select.value].contributors_url;
+      fetchJSONWithPromise(contributorsUrl)
+        .then(contributorsInfo => {
+          removeChildren(listContributor);
+          contributorsInfo.forEach(contributor => {
+            const contributorName = createAndAppend('li', listContributor);
+            contributorName.innerHTML += `<li><a target =_blank href= ${
```
Comment (Meno-101) [inline homework/index.js]: Oops! an `li` inside another `li`.

Comment (Nadoshka) [reply homework/index.js]: U are right! thanks for your feedback! it's done 👍 :)

File: homework/index.js (hunk 1 above)
Comment (Nadoshka) [reply homework/index.js]: Thanks!

**Pull request closed without merging**